src/gas_estimator.rs

```diff
@@ -20,10 +27,10 @@ impl Ord for GasPriceEntry {
     fn cmp(&self, other: &Self) -> Ordering {
         let size1 = &self.sample;
         let size2 = &other.sample;
-        if size1 > size2 {
```

Comment:
If anything other than the GB relayer uses the current GasTracker implementation here in web30, it will behave differently as a result of this change. I observed in my testing only the maximum gas price being selected, rather than the lowest x% as desired.

Reply:
well that was clearly an oversight on my part, this probably needs a basic unit test.

Reply:
Added a test